example/session/src/main/java/org/apache/iotdb/SessionExample.java

```diff
@@ -125,20 +125,19 @@ private static void insertInBatch() throws IoTDBConnectionException, BatchExecut
       valuesList.add(values);
       timestamps.add(time);
       if (time != 0 && time % 100 == 0) {
-        session.insertInBatch(deviceIds, timestamps, measurementsList, valuesList);
+        session.insertRows(deviceIds, timestamps, measurementsList, valuesList);
         deviceIds.clear();
         measurementsList.clear();
         valuesList.clear();
         timestamps.clear();
       }
     }
 
-    session.insertInBatch(deviceIds, timestamps, measurementsList, valuesList);
+    session.insertRows(deviceIds, timestamps, measurementsList, valuesList);
```

Comment:
don't forget to change the method name of ` this. insertInBatch()`

Reply:
fixed